Unified/checkor.py

```diff
@@ -1277,8 +1277,8 @@ def upward( ns ):
 
             if custodial:
                 for output in out_worth_checking:
-                    if size_worth_checking[output] > tape_size_limit:
-                        wfi.sendLog('checkor',"%s output size (%s TB) is too large for the limit set (%s TB)"%( output, size_worth_checking[output], tape_size_limit))
+                    if getDatasetSize(output)/1023 > tape_size_limit:
+                        wfi.sendLog('checkor',"%s output size (%s TB) is too large for the limit set (%s TB)"%( output, out_worth_checking[output], tape_size_limit))
```

Comment:
out_worth_checking is a list, therefore `out_worth_checking[output]` will not function.

`size_worth_checking[output] > tape_size_limit` is a valid expression. I don't see how that can be an issue.

Reply:
size_worth_checking is actually a list, it has to be made into a dict. I thought you guys tested code before merging

Reply:
@vlimant we were getting this issue:
```
failed on cmsunified_task_HIG-RunIISummer19UL16wmLHEGEN-00253__v1_T_200626_230559_5366 due to list indices must be integers, not unicode and Traceback (most recent call last):
File "Unified/checkor.py", line 382, in run
self.check()
File "Unified/checkor.py", line 1280, in check
if getDatasetSize(output)/1023 > tape_size_limit:
TypeError: list indices must be integers, not unicode
```

Reply:
also, `out_worth_checking[output]` is something which was being used earlier as well.

Reply:
https://github.com/CMSCompOps/WmAgentScripts/pull/616/files#diff-7f9486f131994738562992b40266e9ecL1279

Reply:
size_worth_checking has to be turned into a dict. `out_worth_checking[output]` will fail ; guaranteed

Reply:
@vlimant how can one do that. I see size_worth_checking is being defined https://github.com/CMSCompOps/WmAgentScripts/blob/master/Unified/checkor.py#L1078 which is just giving a list.